chapters/adv_tfx/Custom_TFX_Components.ipynb

```diff
@@ -144,6 +145,55 @@
         "\n",
         "![TFX **Component**](https://drive.google.com/uc?export=view&id=1TopCi6XjouwJyVho0PEDIZl9fvEOA1UB)\n",
```

Comment:
I can not access this image, can you access it in an incognito window? Maybe it has private viewing permission.

Reply:
Thank you for the hint/test!